src/django/pfb_analysis/management/commands/update_status.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand
+
+from pfb_analysis.models import AnalysisJobStatusUpdate, AnalysisJob
+
+
+class Command(BaseCommand):
+    help = "Update status during an analysis job"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('job_id')
+        parser.add_argument('status')
+        parser.add_argument('step', nargs='?')
+        parser.add_argument('message', nargs='?')
+
+    def handle(self, *args, **options):
+        try:
+            job = AnalysisJob.objects.get(pk=options['job_id'])
+        except (AnalysisJob.DoesNotExist, ValueError, KeyError):
+            job = None
```

Comment:
See below comment about having null job_ids in the AnalysisJobStatusUpdate model.  Should we instead just let this exception bubble or pass with some other warning rather than inserting a record not attached to a job?